### ISSUE TITLE
Install Script Ends At "Please Point Your Domain At Your ipv6 Address"

### ISSUE THREAD

**Issue by Majoraslayer** [LukeSmithxyz/emailwiz#322]
Title says it all.  The install instructions mention that the script should provide some info that is exported to a file for reference, but instead the script ends with a warning to point my domain at my IPV6 address.  I don't have an IPV6 address, as I've disabled IPV6 on my network for security reasons with my VPN.  I've successfully set this up previously on another server install with Ubuntu, but I'm not having any luck on a new Debian install.  After the script completes, I'm getting a certificate error that seems to be pointing to an invalid internal self-signed certificate using my internal hostname instead of my domain name (according to what Thunderbird is seeing).

Is the script supposed to be pointing the mail server to the correct letsencrypt certificate, or does Debian come with some kind of self-signed certificate I need to delete, and is it failing to do this because of the IPV6 issue?

**Comment by Tair-Karanfili** [comment]
I am also trying to solve this

**Comment by Majoraslayer** [comment]
> I am also trying to solve this

I figured out my own workaround for it, but I'm not sure if it's the "official" way to do it or not.  If you open the install script, there's a section at the top labeled "#Preliminary record checks".  You'll see two lines there for the IPV6 check.  If you place a "#" at the start of both of those lines to comment them out, the script will continue without running the IPV6 check.  After installing this way, I didn't have any issues with the mail server it set up.

I'll leave the issue open though, just in case Luke wants to chime in whether this is the best fix or if he had another way intended to bypass it.

**Comment by Tair-Karanfili** [comment]
@Majoraslayer Everything is working fine for me but some of my mail is going to spam because of this part of my SPF record
"ip6:-all"
Do I get rid of this?

**Comment by Majoraslayer** [comment]
> @Majoraslayer Everything is working fine for me but some of my mail is going to spam because of this part of my SPF record "ip6:-all" Do I get rid of this?

You should be able to.  Here's an example of how my spf record looks, with the domain replaced with an example and my IP replaced with the common address for localhost.  Just swap in your own domain and external IP address:

`v=spf1 mx a:mail.domain.example ip4:127.0.0.1`

**Comment by Tair-Karanfili** [comment]
@Majoraslayer Fixed it. SPF records are valid now

**Comment by LukeSmithxyz** [comment]
Yeah the script assumes you are using both ipv4 and ipv6, you should be able to comment out the line above without major issue.
